Source/SLKTextInputbar.h

```diff
@@ -64,6 +64,8 @@ NS_ASSUME_NONNULL_BEGIN
 @property (nonatomic, readonly) CGFloat appropriateHeight;
 
 
+- (void)slk_updateConstraintConstants;
```

Comment:
This needs to be public so we can rejigger their constraint logic when we need to